src/Router.js

```diff
@@ -98,6 +99,15 @@ class Router extends Component {
   render() {
     if (!this.state.reducer) return null;
 
+    BackAndroid.addEventListener('hardwareBackPress', () => {
+      try {
+        Actions.pop();
+        return true;
+      } catch (err) {
+        return false;
```

Comment:
Why not something like this instead this `return false;`:

```
return typeof this.handleAndroidExit === 'function' ? this.handleAndroidExit() : false;
```

Where the `handleAndroidExit()` is a function passed as a `props` to the `Router`. Like this people would be able to define it's custom exit behavior and/or rely to the default one.

This would do the trick I guess.